src/resmom/mom_main.c

```diff
@@ -6733,7 +6733,11 @@ calc_cpupercent(job *pjob, u_long oldcput, u_long newcput, time_t sampletime)
 		assert(rd != NULL);
 		preswalltime = find_resc_entry(at_used, rd);
 		assert(rd != NULL);
-		wallt = preswalltime->rs_value.at_val.at_long;
+		if (preswalltime != NULL) {
```

Comment:
You could also check preswalltime>rs_value.at_flags & ATR_VFLAG_SET and then only assign the value to wallt.